Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -668,4 +674,32 @@ def test_search
   ensure
     desc_cache.unlink
   end
+
+  def test_bundle
+    skip_unless_test_cmd_taps
+    HOMEBREW_REPOSITORY.cd do
+      shutup do
+        system "git", "init"
+        system "git", "commit", "--allow-empty", "-m", "This is a test commit"
+      end
+    end
+
+    mktmpdir do
+      FileUtils.touch "Brewfile"
+      assert_equal "The Brewfile's dependencies are satisfied.",
+        cmd("bundle", "check")
+    end
+  ensure
+    (HOMEBREW_REPOSITORY/".git").rmtree
+  end
+
+  def test_cask
+    skip_unless_test_cmd_taps
+    assert_equal "Warning: nothing to list", cmd("cask", "list")
+  end
+
+  def test_services
+    skip_unless_test_cmd_taps
+    assert_equal "Warning: No services available to control with `brew services`", cmd("services", "list")
```

Comment:
Can you wrap this overly long line, just like you did in the `bundle check` case?